fakedata-quickcheck.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.34.4.
+-- This file has been generated from package.yaml by hpack version 0.34.2.
```

Comment:
Not exactly required: But do you think you can undo the cabal file changes too ?

Reply:
You're right, no need to do unnecessary change.